package.json

```diff
@@ -1,5 +1,8 @@
 {
   "private": true,
+  "engines": {
+    "node": ">= 18.16 < 19"
```

Comment:
Do we necessarily have to set an upper limit? I think our pre-install script currently checks the lower bound but doesn't do anything for the upper.

Reply:
I think it's common that there are issues with newer versions, that's why I prefer to control it further. This will be a reminder if some user update their node version without realizing that and then report a bug. Althought that reporting a bug can be useful too, as we may know about the problems earlier than when we try it ourselves?
Still I believe we can relax this later if this proves to be more a drawback than an advantage.
What do you think?

Reply:
Oh sorry I just saw this comment. I think it's good like this as well. We will definitely know when we try to upgrade the node version ourselves. We can also change it in the future if people complain about it, but I doubt it :)